libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -142,6 +143,9 @@ void SRV_Channel::aux_servo_function_setup(void)
     case k_egg_drop:
         set_range(100);
         break;
+#if AP_TIE_DOWN_CLAMPS_ENABLED
+    case k_tie_down_release:
```

Comment:
my preference for new options would be to set the range to 1.0, its a float now so the value doesn't matter. Eventually I would like to move everything to 0 -> 1 or -1 -> 1

Reply:
you could also call `set_range` from landing gear of course, that would remove the need for any change in this lib.

Reply:
Done and done